src/ui.rs

```diff
@@ -171,6 +172,18 @@ impl Ui {
         Ok(())
     }
 
+    pub fn write_clap_err_and_exit(&mut self, err: clap::Error) -> ! {
+        // Color parsing must never panic, otherwise we'll never print help.
+        let color = use_color(color_setting(&self.settings));
```

Comment:
You could assign the `color` variable we calculate on line 79 (of this version) to a field in `Ui` so we don't have to re-calculate it here.